Terminal.Gui/Views/Toplevel.cs

```diff
@@ -397,9 +397,7 @@ public virtual void PositionToplevel (Toplevel top)
         }
 
         if ((superView != top || top?.SuperView is { } || (top != Application.Top && top.Modal) || (top?.SuperView is null && top.IsOverlapped))
-
-            // BUGBUG: Prevously PositionToplevel required LayotuStyle.Computed
-            && (top.Frame.X + top.Frame.Width > maxWidth || ny > top.Frame.Y) /*&& top.LayoutStyle == LayoutStyle.Computed*/)
+            && (top.Frame.X + top.Frame.Width > maxWidth || ny > top.Frame.Y))
         {
             if ((top.X is null || top.X is PosAbsolute) && top.Frame.X != nx)
```

Comment:
X can't be null, can it?

I think a nullability pass on this file is probably in order at some point.

Reply:
No. `X` and `Y` cannot be null.

![image](https://github.com/gui-cs/Terminal.Gui/assets/585482/4efdef52-17e3-4b25-9305-a8ae7fb84fd2)